mlir/include/mlir/Dialect/Bufferization/IR/BufferizationTypeInterfaces.h

```diff
@@ -0,0 +1,21 @@
+//===- BufferizationTypeInterfaces.h - Bufferization type interfaces -*- C++
```

Comment:
Nit: 80 cols.

Reply:
I haven't figured how to do it in 80 :| are there any tips when the file name too long? should i drop the middle part?

Reply:
trimmed the middle part, please tell if it's acceptable.